docs/deployments/batch-api/api-configuration.md

```diff
@@ -20,7 +20,7 @@ Reference the section below which corresponds to your Predictor type: [Python](#
     env: <string: string>  # dictionary of environment variables
   networking:
     endpoint: <string>  # the endpoint for the API (default: <api_name>)
-    api_gateway: public | none  # whether to create a public API Gateway endpoint for this API (if not, the load balancer will be accessed directly) (default: public)
+    api_gateway: public | none  # whether to create a public API Gateway endpoint for this API (if not, the load balancer will be accessed directly) (default: public, unless disabled cluster-wide)
```

Comment:
`(if not, the load balancer will be accessed directly)` -> `(if not, this API will be accessible via the load balancer)`